doc/nrf/ug_gzll.rst

```diff
@@ -70,7 +65,7 @@ The radio and timer interrupt handlers should run at priority level 0 (highest p
 To avoid blocking Gazell operations, applications can run at priority level 2 or higher.
 
 You can customize Gazell at runtime for a range of different applications.
-See the :ref:`nrfxlib:gzll_api` for a list of configuration functions as well as the default and constant parameters.
+See the :ref:`nrfxlib:gzll` and :ref:`nrfxlib:gzll_api` for a list of configuration functions as well as the default and constant parameters.
 
 .. note:
    Editing the header file containing the default and constant parameters does not change their value when compiling a new project.
```

Comment:
Not sure what this is, and if it belongs here? If it does, let me know because now it is a comment in the .rst file and not visible in the html-build.

Reply:
If the note is needed, you at least need to have another colon after the word "note" -> .. note::